eval.py

```diff
@@ -4,77 +4,195 @@
     CropHarvestEval("Togo", ignore_dynamic_world=True),
     CropHarvestEval("Brazil", ignore_dynamic_world=True),
+    CropHarvestEval("Kenya", ignore_dynamic_world=True, seed=0),
+    CropHarvestEval("Togo", ignore_dynamic_world=True, seed=0),
+    CropHarvestEval("Brazil", ignore_dynamic_world=True, seed=0),
+    CropHarvestEval("Kenya", ignore_dynamic_world=True, seed=84),
+    CropHarvestEval("Togo", ignore_dynamic_world=True, seed=84),
+    CropHarvestEval("Brazil", ignore_dynamic_world=True, seed=84),
+    FuelMoistureEval(),
+    AlgaeBloomsEval(),
+    FuelMoistureEval(seed=0),
+    AlgaeBloomsEval(seed=0),
+    FuelMoistureEval(seed=84),
+    AlgaeBloomsEval(seed=84),
+    # no seeds for EuroSat, which we evaluate using
+    # a KNN classifier
+    EuroSatEval(rgb=True, input_patch_size=32),
+    EuroSatEval(rgb=True, input_patch_size=16),
+    EuroSatEval(rgb=True, input_patch_size=8),
+    EuroSatEval(rgb=True, input_patch_size=4),
+    EuroSatEval(rgb=True, input_patch_size=2),
+    EuroSatEval(rgb=True, input_patch_size=1),
+    EuroSatEval(rgb=False, input_patch_size=32),
+    EuroSatEval(rgb=False, input_patch_size=16),
+    EuroSatEval(rgb=False, input_patch_size=8),
+    EuroSatEval(rgb=False, input_patch_size=4),
+    EuroSatEval(rgb=False, input_patch_size=2),
+    EuroSatEval(rgb=False, input_patch_size=1),
+    TreeSatEval("S1", input_patch_size=1),
+    TreeSatEval("S2", input_patch_size=1),
+    TreeSatEval("S1", input_patch_size=2),
+    TreeSatEval("S2", input_patch_size=2),
+    TreeSatEval("S1", input_patch_size=3),
+    TreeSatEval("S2", input_patch_size=3),
+    TreeSatEval("S1", input_patch_size=6),
+    TreeSatEval("S2", input_patch_size=6),
     CropHarvestEval("Kenya"),
     CropHarvestEval("Togo"),
     CropHarvestEval("Brazil"),
+    CropHarvestEval("Kenya", seed=0),
+    CropHarvestEval("Togo", seed=0),
+    CropHarvestEval("Brazil", seed=0),
+    CropHarvestEval("Kenya", seed=84),
+    CropHarvestEval("Togo", seed=84),
+    CropHarvestEval("Brazil", seed=84),
 ]
-eval_task_list.extend(
-    [CropHarvestEval("Togo", ignore_dynamic_world=True, num_timesteps=x) for x in range(1, 12)]
-)
-eval_task_list.extend(
-    [CropHarvestEval("Kenya", ignore_dynamic_world=True, num_timesteps=x) for x in range(1, 12)]
-)
 # add CropHarvest over time
+for seed in [DEFAULT_SEED, 0, 84]:
+    eval_task_list.extend(
+        [
+            CropHarvestEval("Togo", ignore_dynamic_world=True, num_timesteps=x, seed=seed)
+            for x in range(1, 12)
+        ]
+    )
+    eval_task_list.extend(
+        [
+            CropHarvestEval("Kenya", ignore_dynamic_world=True, num_timesteps=x, seed=seed)
+            for x in range(1, 12)
+        ]
+    )
+
+if wandb_enabled:
+    eval_config = {
+        "model": model.__class__,
+        "encoder": model.encoder.__class__,
+        "decoder": model.decoder.__class__,
+        "device": device,
+        "model_parameters": "random" if fully_supervised else path_to_state_dict,
```

Comment:
I think I should have added `"logging_dir": logging_dir,` here but I forgot, same in train.py. Because now `output_dir` gets logged but it's basically always something like `/network/scratch/<u>/<user>/presto/` and then you don't know what the actual subdir for this run was

Reply:
added!